### DISCUSSION TITLE
Gradient color based on values

### DISCUSSION THREAD

**Discussion by bariscope** [bozdoz/wp-plugin-leaflet-map#194] [Q&A]
Hi bozdoz and other contributors, I am embedding a geojson in a wordpress site using this plugin. I would like to show a certain value using color gradient for each polygon in the map (for example: there is a feature called "Price" in the data. I would like to show the range of prices using a defined gradient). Is this possible? What is the way of achieving this? An example of what I am looking for: https://en.wikipedia.org/wiki/Choropleth_map

**Comment by bozdoz** [comment]
Looks like you could add this leaflet plugin to get it to work: 
https://github.com/timwis/leaflet-choropleth

**Reply by bariscope** [reply]
Thank you, this is what will work for me! However, this is an standalone plugin. Any ways to integrate it with the wp-plugin-leaflet-map?

**Reply by bozdoz** [reply]
Yes, check out the FAQ in the README
